Asterisk.2013/Asterisk.NET/Manager/Event/CoreShowChannelEvent.cs

```diff
@@ -0,0 +1,75 @@
+using System;
+
+namespace AsterNET.Manager.Event
+{
+	/// <summary>
+	/// Raised when a Queue member's status has changed
+	/// </summary>
+	public class CoreShowChannelEvent : AbstractChannelEvent
+	{
+		/// <summary>
+		/// Get/Set Channel
+		/// </summary>
+		public string Channel { get; set; }
+
+		/// <summary>
+		/// Get/Set Channel
+		/// </summary>
+		public string Language { get; set; }
+
+		/// <summary>
+		/// Get/Set Context
+		/// </summary>
+		public string Context { get; set; }
+
+		/// <summary>
+		/// Get/Set Exten
+		/// </summary>
+		public string Exten { get; set; }
+		public string Extennnnnnnnnnsions { get; set; }
```

Comment:
What do you mean?

Reply:
This is a test atavism :)
I`ve corrected it. Thanks!